gammapy/estimators/tests/test_ts_map.py

```diff
@@ -119,11 +119,11 @@ def test_compute_ts_map_psf(fermi_dataset):
 
     assert_allclose(result["ts"].data[29, 29], 835.140605, rtol=1e-2)
     assert_allclose(result["niter"].data[29, 29], 7)
-    assert_allclose(result["flux"].data[29, 29], 1.626651e-09, rtol=1e-2)
```

Comment:
Do you understand, why the numbers here changed?

Reply:
This is due to the issue of the flux normalisation - the previous `estimate_exposure` was computing the flux integrated flux between `energy_true.min()` and `energy_true_max()` respectively, whereas presently I compute it for each reco energy bin.

Reply:
Makes sense...however I'm still wondering whether we do the right thing, see https://github.com/gammapy/gammapy/pull/2994#discussion_r477248973t. Let's  wait for @registerrier to join the discussion...